govcd/edgegateway.go

```diff
@@ -186,6 +186,121 @@ func (eGW *EdgeGateway) RemoveNATPortMapping(natType, externalIP, externalPort s
 
 }
 
+// AddDNATRule creates firewall DNAT rule and return refreshed existing NAT rules array or error
+func (eGW *EdgeGateway) AddDNATRule(networkHref, externalIP, externalPort, internalIP, internalPort, protocol, icmpSubType, description string) ([]*types.NatRule, error) {
```

Comment:
I have two problems with this function:
(1) Given that the title of this PR says allow creation of rules for external networks, why is this a method of edge gateway and not the network itself?
(2) I know it's a matter for discussion, but I don't see how passing 8 arguments by position is any better than passing a dedicated structure.

Reply:
(1) Explicitly state in Title and description - this Edge gtw configuration part
(2) Yes, I'm keep inline with existing ones. Cause to many args passed I will update to use Struct

Reply:
(2) change added